examples/web/jobs.html

```diff
@@ -65,7 +65,7 @@
         padding: 0.25rem;
       }
 
-      .example-TextField-grid {
+      .example-TextField-grid, .example-CheckboxField-grid {
```

Comment:
Please break this to new line too

Reply:
Or maybe, can we do it like this?
```
.example-TextField-grid, 
.example-CheckboxField-grid {
        display: grid;
        grid-template-columns: 25rem;
        gap: 1.5rem;
}
.example-TextField-grid {
        gap: 1.5rem;
}
```
Same for index.html please. We don't need the gap in Checkbox grid, because they have the block spacing.

Reply:
Fixed.